EnglishFilter/sections/foreign.txt

```diff
@@ -807,6 +807,11 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 !---------------------------------------------------------------------
 ! https://github.com/AdguardTeam/AdguardFilters/issues/41555
 ||m.pianba.tv/ad/
+! https://github.com/AdguardTeam/AdguardFilters/issues/41968
+technews.tw#%#AG_abortInlineScript('/navigator.userAgent/', 'jQuery');
```

Comment:
I think it should be without `'` - `('/navigator.userAgent/',` => `(/navigator.userAgent/,` and it's regexp, so I think we can escape dot:
`technews.tw#%#AG_abortInlineScript(/navigator\.userAgent/, 'jQuery');`